app/login/page.tsx

```diff
@@ -47,24 +47,29 @@ const LoginPage: React.FC = () => {
             });
         }
 
-        /* eslint-disable @typescript-eslint/no-explicit-any */
-        try {
-            const response = await loginRequest(csrfToken);
+        if (csrfToken) {
```

Comment:
I added a check in both the /login and /signup pages for a csrfToken. If no token is available in the header, then I put a placeholder message of "Something went wrong" that will display on the page.